docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -46,7 +46,12 @@ The `./config/database.js` (or `./config/database.ts` for TypeScript) accepts 2
 | `acquireConnectionTimeout`<br /><br />_Optional_         | How long knex will wait before throwing a timeout error when acquiring a connection (in milliseconds) | `Integer` | `60000` |
 
 :::note
-A `client` value of 'sqlite' will be modified by Strapi to be 'better-sqlite3' if the package is available in your project, or 'sqlite3' if it is not.
+Strapi only supports the following client values, and will automatically rewrite the 'client' value to the following options before passing the configuration to Knex
+
+| client       | actual package used |
```

Comment:
```suggestion
| client       | actual package used |
|----------|----------------------|
```

I haven't double-checked, but I fix it should fix the following rendering issue:

![Screenshot 2023-10-19 at 13 57 58](https://github.com/strapi/documentation/assets/4233866/f1e58f3e-8e4e-4ad9-9dd3-c68fab853ed2)

Reply:
ahh, apparently we can't nest a table (or I did it incorrectly)

What do you think we should do here?

Reply:
Have you tried adding the line I suggested? 😊 

I believe it's more because you're missing the "dotted" line between headers and cells rather than not being able to nest a table inside a callout (improper Markdown table syntax rather than Docusaurus limitation)
![Screenshot 2023-10-19 at 16 56 05](https://github.com/strapi/documentation/assets/4233866/64822931-d400-4e1f-88d5-2d30b06be424)